scripts/zmk_local_install.sh

```diff
@@ -15,17 +15,17 @@ sudo apt-get install --yes --no-install-recommends git cmake ninja-build gperf \
 # Find latest release version
 # ZSDK_URL="https://github.com/zephyrproject-rtos/sdk-ng/releases/latest"
 # ZSDK_VERSION="$(curl -fsSLI -o /dev/null -w %{url_effective} ${ZSDK_URL} | sed 's/^.*v//')"
-ZSDK_VERSION="0.15.2"
+ZSDK_VERSION="0.16.5-1"
```

Comment:
Let's stick to the version officially [supported by ZMK](https://zmk.dev/docs/development/setup/native#4-install-zephyr-sdk)

```suggestion
ZSDK_VERSION="0.16.3"
```